components/automate-dex/habitat/plan.sh

```diff
@@ -39,7 +39,7 @@ do_before() {
 do_unpack() {
   git clone "https://github.com/chef/dex-1" "$GOPATH/src/github.com/chef/dex-1"
   ( cd "$GOPATH/src/github.com/chef/dex-1" || exit
-    git checkout add_invalid_attempts
+    git checkout sanju/test_refresh
```

Comment:
raise a pr from dex-1 against add_invalid_attempts from sanju/test_refresh mention that pr in conversation

Reply:
okay

Reply:
https://github.com/chef/dex-1/pull/81